features/step_definitions/generator_steps.rb

```diff
@@ -28,6 +35,11 @@
   EOH
 end
 
+Given(/^I debug$/) do
```

Comment:
I'd rather not add this if it means adding `pry` as a dep, but will defer to others' opinion.